samples/IISSample/web.config

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
Because we use this as a reference to show people what it should look like, and so it's easier to modify when testing.